src/getBlobFromDataUri.ts

```diff
@@ -0,0 +1,25 @@
+// https://stackoverflow.com/questions/12168909/blob-from-dataurl
```

Comment:
Can you check if we can use a package like [`data-uri-to-buffer`](https://www.npmjs.com/package/data-uri-to-buffer)?

Parsing data URI is tricky and the code from SO is not handling all cases.

Reply:
done, but I've only lightly tested it because I'm no longer working on the project where this library is required. I recommend more thorough testing.

Reply:
The change looks reasonable - I'd be more comfortable with the following -
1. All tests pass
2. Manual tests using other browsers.